experimenter/experimenter/nimbus-ui/src/components/PageSummary/Takeaways/index.test.tsx

```diff
@@ -259,32 +294,6 @@ describe("TakeawaysEditor", () => {
     expect(result2.takeawaysMetricGain).toEqual(true);
   });
 
-  it("submits takeaway checkbox values even if value not changed", async () => {
```

Comment:
This case is already covered by "submits form data when save is clicked"

Reply:
I love removing duplicate tests 🎉